docs/proposals/csi-webhook.md

```diff
@@ -0,0 +1,71 @@
+# Proposal: Validation webhook for CSI Deployment
+
+Last updated: 01.09.2021
+
+
+## Abstract
+
+Add CSI Deployment Validation webhook in csi-baremetal Operator
+
+## Background
+
+Need to check that there is only one csi-baremetal instance on the cluster. 
```

Comment:
I think the common idea is that we control CR create/update operations. It can be used for multiple purposes - for example, checking upgrade version matrix

Reply:
Fixed